.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+<!-- Write a description of what the PR solves and how -->
+
+<!-- Link to relevant Jira issue, add multiple if necessary -->
+Jira Issue: [RHELC-]()
+
+Checklist
+- [ ] PR meets acceptance criteria specified in the Jira issue
+- [ ] PR has been tested manually in a VM (either author or reviewer)
+- [ ] Jira issue has been made public if possible
+- [ ] `[RHELC-]` is part of the PR title <!-- For a proper sync with Jira -->
+- [ ] Code and tests are documented properly
+- [ ] The commits are squashed to as few commits as possible (without losing data) <!-- The commits can be squashed to 1 commit, but then we might lose data regarding moving something to a new file and then refactoring for example. Hence squash without losing data -->

```

Comment:
I would place a short link to some page that shows people how to squash commits. This might not be as straight forward to people as we may think it is.